resources/documentation/deploy_bluebanquise.rst

```diff
@@ -192,8 +212,8 @@ SSH public key
 
 In order to log into the remote nodes without giving the password, check that
 the ssh public key defined in authentication.yml in your inventory match your
-management1 public key. If not, update it. Remember to run the pxe_stack role
-after updating the configuration.
+management1 public key. If not, update it. Remember to run again the pxe_stack
```

Comment:
Do you mean one need to run the pxe_stack role twice after updating the configuration?

Reply:
Updated, should be ok now.